src/modules/game.js

```diff
@@ -82,6 +90,13 @@ class Game {
 
     gameOver() {
         this.gameState = GAMESTATE.GAMEOVER;
+        gamePage.style.display = 'none';
+        gameOverPage.style.display = 'flex';
+        this.enemies.forEach((enemy) => {
+            enemy.delete();
+        });
+        this.answerInput.value = '';
+        enemySpeed = 40;
```

Comment:
I think we need to put all the game reset logic in its own method, ive tested it and everything is being reset at the moment apart from the spawn timer.